tests/python_tests/utils_longbench.py

```diff
@@ -0,0 +1,245 @@
+import re
```

Comment:
This code is probably taken from 3rd party repo. We need to keep the license in this case.